common/app/views/fragments/amp/stylesheets/liveBlog.scala.html

```diff
@@ -207,3 +207,108 @@
     left: 0;
     right: 0;
 }
+
+/* Pagination
+========================================= */
+
+.liveblog-navigation {
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    font-family: "Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;
+    display: flex;
+    padding-bottom: .75rem;
+    margin: 0 20px;
+}
+
+.liveblog-navigation__detail {
+    text-align: center;
+    order: 1;
+    flex: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.liveblog-navigation__newer {
+    float: left;
+}
+
+.liveblog-navigation__older {
+    float: right;
+    order: 2;
+}
+
+.liveblog-navigation__link {
+    display: inline-block;
+    position: relative;
+    color: #333;
+    border: 0.0625rem solid #dcdcdc;
+    height: 2.25rem;
+    line-height: 2.375rem;
+    font-weight: bold;
```

Comment:
Because it is a different font, I am not sure if bold looks the best with this design.